cpp/src/iCubModels.cpp.in

```diff
@@ -0,0 +1,35 @@
+/**
+ * @file iCubModels.cpp
+ * @authors Giulio Romualdi
+ * @copyright 2022 Istituto Italiano di Tecnologia Released under the terms of the Creative Commons
+ * Attribution Share Alike 4.0 International
+ */
+
+#include <iCubModels/iCubModels.h>
+
+namespace iCubModels
+{
+std::string getModelsPath()
+{
+    return "@ICUB_MODELS_MODELS_PATH@";
```

Comment:
I remember that in one case conda has problem in relocating the string, I want to double check that we use the variant that does not create problems here.

Reply:
I think we should be safe here (the problematic case is the one using preprocessor macros, see https://github.com/conda-forge/libignition-gazebo-feedstock/issues/4 and https://github.com/conda/conda-build/issues/1674 . However, just to be safe I suggest a small modification:
```suggestion
    std::string retstr = "@ICUB_MODELS_MODELS_PATH@";
    return retstr;
```